apps/src/music/views/MusicPlayView.tsx

```diff
@@ -0,0 +1,101 @@
+import React, {memo, useCallback, useMemo} from 'react';
+
+import {Button} from '@cdo/apps/componentLibrary/button';
+import {Heading2, BodyTwoText} from '@cdo/apps/componentLibrary/typography';
+import moduleStyles from './music-play-view.module.scss';
+import Link from '@cdo/apps/componentLibrary/link';
+
+import musicI18n from '../locale';
+
+const codeOrgLogo = require(`@cdo/static/music/code-dot-org-white-logo.png`);
+
+interface MusicPlayViewProps {
+  onPlay: () => void;
+  onStop: () => void;
+  isPlaying: boolean;
+}
+
+const MusicPlayView: React.FunctionComponent<MusicPlayViewProps> = ({
+  isPlaying,
+  onPlay,
+  onStop,
+}) => {
+  const shareData = useMemo(
+    () => ({
+      title: 'Project Name',
+      url: window.location.href,
+    }),
+    []
+  );
+  // Share button will only appear when users browser supports the Web Share API (Can be mobile browsers and some desktop browser like macOS Safari)
+  // Requires HTTPS connection (adhoc or production page).
+  // Was unable to access it on localhost with our routing (http://localhost-studio.code.org:3000/
+  // is not considered a sage URL by browser, only http://localhost:3000/ is).
+  const canShare =
+    navigator && navigator.canShare && navigator.canShare(shareData);
+  const shareProject = useCallback(() => {
+    navigator?.share(shareData);
+  }, [shareData]);
+
+  return (
+    <div className={moduleStyles.musicPlayViewContainer}>
+      <div className={moduleStyles.musicPlayViewCard}>
+        <img src={codeOrgLogo} alt="Code.org" />
+        <Heading2>
+          {/*TODO: Project name should be here instead of placeholder text*/}
+          My Awesome Mix
+        </Heading2>
+        <BodyTwoText>
+          {musicI18n.madeWithMusicLabOn()}{' '}
+          <Link href="https://studio.code.org/projects/music">Code.org</Link>
```

Comment:
This should use a relative link so that it goes to the right page in various environments.  That said, should it perhaps just go to our homepage since the text is "Code.org"?